Add interactive graphic showing zdm migration phases in one presentat… (#147)
datastax/migration-docs: docs-src/zdm-core/modules/migrate/partials/interactive-lab.adoc

> Now that you've seen a conceptual overview of the process, why not putting what you learned in practice? 

Reviewer: ```suggestion
Now that you've seen a conceptual overview of the process, why not put what you learned into practice? 
```

Other options:
- "Now that you've seen a conceptual overview of the process, let's put what you learned into practice."
- "Now that you've seen a conceptual overview of the process, you can try putting what you learned into practice."

Author: Done in latest commit, pushed, and rebuilt on coppi. Thanks Eric.

We've built a complementary learning resource that is a companion to this comprehensive ZDM documentation. It's the {zdm-product} Interactive Lab, available for you here:

> link:https://www.datastax.com/dev/zdm[https://www.datastax.com/dev/zdm, window="_blank"]

Reviewer: ```suggestion
https://www.datastax.com/dev/zdm[,window="_blank"]
```

Author: I like the conversational "Now that you've..." line but will make a minor edit.  I'll also change the L5 link as you proposed (thanks).  That will come in the next commit. Resolving for now without this commit.

Author: Updated in latest commit.
